_posts/2021-01-04-migrating-from-prestosql-to-trino.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Migrating from PrestoSQL to Trino
+author: David Phillips
+excerpt_separator: <!--more-->
+---
+
+As we previously announced, we're
+[rebranding Presto SQL as Trino]({% post_url 2020-12-27-announcing-trino %}).
+Now comes the hard part: migrating to the new version of the software.
+We just released the first version,
+[Trino 351](/docs/current/release/release-351.html),
+which uses the name Trino everywhere, both internally and externally.
+Unfortunately, there are various unvoidable compatibility aspects that
+administrators of Trino need to know about. We hope this post explains
+them well and makes the transition as smooth as possible.
+
+<!--more-->
+
+# Things that haven't changed
+
+Let's start with the good news. For end users running queries against Trino,
+everything should be the same. There are no changes to the SQL language,
+SQL functions, session properties, etc.
+
+Users will now see *Trino* in error messages, and error stack traces will have
+a different package name, but otherwise they won't know that anything has changed.
+All of their views, reports, or other stored queries will work as before.
+
+Similarly for administrators, except for a few things noted in the
+[Trino 351 release notes](/docs/current/release/release-351.html),
+all the configuration properties are the same.
+
+# Client protocol compatiblity
+
+The client protocol is how clients like the
+[CLI](docs/current/installation/cli.html) or
+[JDBC driver](/docs/current/installation/jdbc.html)
+talk to Trino. It uses standard HTTP as the underlying communications
+protocol, with some custom HTTP headers to communicate values
+to and from Trino. Unfortunately, those header names started with
+`X-Presto-` and thus had to be changed to `X-Trino-`.
+
+The Trino CLI and JDBC driver send the new headers, so they are
+**only compatible with Trino versions 351 and newer**. Users should
+thus wait to upgrade the CLI or JDBC driver until the Trino servers
+they talk to have been upgraded.
+
+Out of the box, the Trino server does not work with older clients.
+However, in order to support a graceful transition, you can allow the
+server to support older clients by adding a configuration property:
+
+```
+protocol.v1.alternate-header-name=Presto
+```
+
+**We recommend using version 350 of CLI and JDBC driver as the transition version**,
```

Comment:
End sentence and start new one (and use comma there)

versions. It has all the newest features, such as ..